frontend/src/metabase/components/ModalContent/ModalContent.tsx

```diff
@@ -17,6 +28,9 @@ interface CommonModalProps {
   // standard modal
   formModal?: boolean;
   centeredTitle?: boolean;
+
+  headerActions?: ModalHeaderAction[];
```

Comment:
I've seen many places like this with the `actions` prop where it was just `ReactNode`, maybe we should do the same here.